plugins/modules/one_image.py

```diff
@@ -20,42 +20,65 @@
   - community.general.attributes
 attributes:
-  check_mode:
-    support: full
-  diff_mode:
-    support: none
+    check_mode:
+        support: full
+    diff_mode:
+        support: none
 options:
-  id:
-    description:
-      - A O(id) of the image you would like to manage.
-    type: int
-  name:
-    description:
-      - A O(name) of the image you would like to manage.
-    type: str
-  state:
-    description:
-      - V(present) - state that is used to manage the image
-      - V(absent) - delete the image
-      - V(cloned) - clone the image
-      - V(renamed) - rename the image to the O(new_name)
-    choices: ["present", "absent", "cloned", "renamed"]
-    default: present
-    type: str
-  enabled:
-    description:
-      - Whether the image should be enabled or disabled.
-    type: bool
-  new_name:
-    description:
-      - A name that will be assigned to the existing or new image.
-      - In the case of cloning, by default O(new_name) will take the name of the origin image with the prefix 'Copy of'.
-    type: str
-  persistent:
-    description:
-      - Whether the image should be persistent or non-persistent.
-    type: bool
-    version_added: 9.5.0
+    id:
+        description:
+            - A O(id) of the image you would like to manage.
+        type: int
+    name:
+        description:
+            - A O(name) of the image you would like to manage.
+            - Required if O(create=true).
+        type: str
+    state:
+        description:
+            - V(present) - state that is used to manage the image
+            - V(absent) - delete the image
+            - V(cloned) - clone the image
+            - V(renamed) - rename the image to the O(new_name)
+        choices: ["present", "absent", "cloned", "renamed"]
+        default: present
+        type: str
+    enabled:
+        description:
+            - Whether the image should be enabled or disabled.
+        type: bool
+    new_name:
+        description:
+            - A name that will be assigned to the existing or new image.
+            - In the case of cloning, by default O(new_name) will take the name of the origin image with the prefix 'Copy of'.
+        type: str
+    persistent:
+        description:
+            - Whether the image should be persistent or non-persistent.
+        type: bool
+        version_added: 9.5.0
+    create:
+        description:
+            - Whether the image should be created if not present.
+        type: bool
+        version_added: 10.0.0
+    template:
+        description:
+            - Use with O(create=true) to specify image template.
+        type: str
+        version_added: 10.0.0
+    datastore_id:
+        description:
+            - Use with O(create=true) to specify datastore for image.
+        type: int
+        version_added: 10.0.0
+    wait_timeout:
+        description:
+            - Seconds to wait until image is ready, deleted or cloned.
+        type: int
+        required: false
```

Comment:
`required=false` is the default:
```suggestion
```

Reply:
my bad :) i thought you should specify `required: false` if default value is provided (the same way it should be in the argument_spec)